src/style.css

```diff
@@ -1,3 +1,80 @@
+:root {
+  --colors-link: rgba(0, 172, 220, 1);
+}
+
 body {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  font-family: Roboto, Arial, Helvetica, serif;
+  font-weight: 500;
+  font-style: normal;
+  font-size: 12px;
+  line-height: 14px;
+  text-transform: uppercase;
```

Comment:
The `font-size` and `line-height` are set to relatively small values. Ensure that this is intentional and that the text remains readable on all devices. Consider using relative units like `em` or `rem` for better scalability.